### DISCUSSION TITLE
My application to GitHub Education for students is not approved

### DISCUSSION THREAD

**Discussion by dacistul** [community/community#142974] [GitHub Education]
### Hi everyone,

I'm student from Romania and the Student ID has two faces, so I've taken photos of both and put them together in the same picture. I've applied to 4 times till now and is not even funny anymore. I have 2-step activated and in the billing info I have my name(same as in the ID) and address. Also, the faculty email account has a verified domain.

I don't know what to do anymore

**Comment by aymena80s545** [comment]
https://t.me/Helloo_FRIENDD
